codacy/values-production.yaml

```diff
@@ -66,22 +66,22 @@ codacy-ingress:
   create: true
   ingress:
     hostname:
-      app: <--- codacy-app.dns.internal ---> # Codacy application DNS 
-      api: <--- codacy-api.dns.internal ---> # Codacy API DNS
+      app: <--- codacy-app.dns.internal ---> # Codacy application hostname
+      api: <--- codacy-api.dns.internal ---> # Codacy API hostname
     annotations:
       kubernetes.io/ingress.class: nginx
       kubernetes.io/tls-acme: "true"
       certmanager.k8s.io/acme-challenge-type: http01
       certmanager.k8s.io/cluster-issuer: letsencrypt
-      external-dns.alpha.kubernetes.io/hostname: '<--- codacy-app.dns.internal --->, <--- codacy-api.dns.internal --->' # Codacy application and API DNS 
+      external-dns.alpha.kubernetes.io/hostname: '<--- codacy-app.dns.internal --->, <--- codacy-api.dns.internal --->' # Codacy application and API hostnames
```

Comment:
Is "hostnames" the correct term here? Or should it be "domain names"?

Reply:
If we remove this we need to add to the docs how to get the IP/address of the ingress for people to setup their DNS.
@joaocosta-codacy can you help build a command that gets this value from the ingress?